cmd/krew/cmd/install.go

```diff
@@ -135,7 +139,7 @@ Remarks:
 				internal.PrintSecurityNotice()
 			}
 			if len(failed) > 0 {
-				return errors.Errorf("failed to install some plugins: %+v", failed)
+				return errors.Wrapf(returnErr,"failed to install some plugins: %+v", failed)
```

Comment:
Why not simply
```suggestion
				return returnErr
```

Each failed plugin is already printed in line 128.

Reply:
I'm thinking we can just wrap as `plugin install error` wdyt?

wrapping helps a lot as we've seen in other bug reports.

Reply:
I mean yes, there's quite a bit of overlap with 128.
maybe we should unwrap the error in 128 to its root cause, im not sure.

Reply:
Then let's keep the current version. When installing multiple plugins, this shows a full list of failed plugins which might help.

Reply:
Yeah the worst part of this is most people don't install multiple plugins at a time, but this makes the error message 2x longer by duplicating a long error message for everybody.

Reply:
:thinking: But they should see a long stack-trace too, right? Basically, I'm ok with both options. Your call :)

Reply:
I'm not too worried about not having stack trace (%+v) at line 128 –again, since, most users install plugins one-by-one and we have decent error-wrapping.